functions/_fzf_search_completions.fish

```diff
@@ -0,0 +1,189 @@
+# External limitations:
+# - fish-shell/fish-shell#9577: folders ending in '=' break fish's internal path completion
+#   (It thinks that it has to complete from / because `a_folder=/` looks like an argument to fish.
+# - fish cannot give reliable context information on completions. Knowing whether a completion
+#   is a file or argument can only be determined via a heuristic.
+# - Completion is slow. Not because this script is slow but because fish's `complete` command
+#   can take several seconds even for just 100 entries. Just run `time complete --escape --do-complete l`
+#   to see for yourself.
+set --global _fzf_search_completions_min_description_offset 14
+function _fzf_search_completions --description "Shell completion using fzf"
+	# Produce a list of completions from which to choose
+	# and do nothing if there is nothing to select from
+	set -l cmd (commandline -co) (commandline -t)
```

Comment:
could you use the long version of these flags please? `--cut-at-cursor --tokenize`, `--current-token`. Btw, how come you need the second `(commandline -t)`?

Reply:
Good suggestion, will do. Regarding your question, consider `ls -l --color=<TAB>`
- `commandline --cut-at-cursor --tokenize` will yield `ls` and `-l` (only tokens before the cursor)
- `commandline --current-token` yields `--color=` (the current token until the cursor)